src/components/TopBar.js

```diff
@@ -206,7 +207,7 @@ export default class TopBar extends Component {
     return div({ style: styles.topBar }, [
       icon('bars', {
         size: 36,
-        style: { marginRight: '2rem', color: colors.purple[0], cursor: 'pointer' },
+        style: { marginRight: '2rem', color: colors.purple[0], minWidth: 36, cursor: 'pointer' },
```

Comment:
FYI: Another way to solve this would be to add `flex: none` so it can't collapse.